packages/oscal-react-library/src/components/OSCALCatalogBaseline.tsx

```diff
@@ -36,19 +36,21 @@ import { FormatBold, FormatItalic, FormatQuote, Subscript, Superscript } from "@
 
 import { CodeOffSharp } from "@mui/icons-material";
 
+import GroupDrawer from "./OSCALCatalogManageGroup";
+
 const Hug = styled(Container)`
   position: absolute;
   width: 348px;
   height: 36px;
   top: 120px;
-  left: 810px;
+  left: 840px;
```

Comment:
You don't have to make the change here, but in the future can we start using `rem` instead of `px` for our style sizing? Emily made this choice in the styleguide and I would like to follow this method. This is useful if we were to ever change our default font size, everything else will scale correctly.

`1rem` is equivalent to our font size and our font size is set to `16px`. I mostly try to think in divisible of `4px` when sizing (what is the unit of measure Emily mentioned in the grid design of the application). Here's a quick idea of how `px` sizes translate to `rem`:
- `4px` - `0.25rem`
- `8px` - `0.5rem`
- `12px` - `0.75rem`
- `16px` - `1rem`
- `24px` - `1.5rem`
- `32px` - `2rem`
- `64px` - `4rem`